project/src/main/webapp/profile.js

```diff
@@ -32,24 +32,30 @@ function updateProfileEvents() {
 
 async function updateUserEventsHosting() {
   const userEventsHosting = await getUserEvents('hosting');
-  for (const eventHosting of userEventsHosting) {
-    populateEventContainer(eventHosting, 'events-hosting');
+  for (const eventsKey in userEventsHosting) {
```

Comment:
Are you still able to use for/of here like before? If so, that seems to be more aligned w/ the JS style guide

Otherwise, LGTM!

Reply:
Unable to because unfortunately the data is in dictionary style.